### PR TITLE
demand type can be case insenstive

### PR DESCRIPTION
**Pull request by longshuicy**

### CODE REVIEW

**Comment by longshuicy** [comment]
run pytest `test_plot_fragility(client)`

File: tests/pyincore_viz/data/StandardFragilityCurveDemandType.json [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    "resultType": "Limit State",
+    "hazardType": "earthquake",
+    "inventoryType": "building",
+    "creator": "incore",
+    "curveParameters": [
+        {
+            "name": "zero_point_two_sec_Sa",
+            "unit": "g",
+            "description": "0.2 sec Sa value from hazard service",
+            "fullName": "0.2 SEC SA",
```
Comment (longshuicy) [inline tests/pyincore_viz/data/StandardFragilityCurveDemandType.json:18]: now it's case insensitive

**Review by ywkim312 — APPROVED** [review]

**Comment by navarroc** [comment]
Looks good. Are the pytests fixed on develop? It would be good to merge in develop if that's the case so we verify the tests.

**Comment by ywkim312** [comment]
> Looks good. Are the pytests fixed on develop? It would be good to merge in develop if that's the case so we verify the tests.

yes pytests were fixed. I will merge and check if the pytests works okay in github action.

**Review by navarroc — APPROVED** [review]

**Pull request merged**